doc/api/async_hooks.md

```diff
@@ -304,7 +304,9 @@ In the case of Promises, the `resource` object will have `promise` property
 that refers to the Promise that is being initialized, and a `parentId` property
 set to the `asyncId` of a parent Promise, if there is one, and `undefined`
 otherwise. For example, in the case of `b = a.then(handler)`, `a` is considered
-a parent Promise of `b`.
+a parent Promise of `b`. Note that, in the cases where `parentId` is defined, it
```

Comment:
Nit: Get rid of `Note that, in the cases where` and replaced `If` or `When` so the sentence might be:

> If`parentId` is defined, it...

Reply:
Done.